src/objects/rhythm/rhythm.stories.mdx

```diff
@@ -8,132 +9,125 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 
-`o-rhythm--compact` is best for shorter content chunks that just need a _pinch_ more space:
+- `compact`: For short chunks of summary content, such as a heading, sentence and button.
+- `condensed`: For short lists of content that need a bit of space between.
+- `default`: Reset to the default amount of space, useful for nested rhythm objects.
+- `generous`: Apply enough space for each element to feel distinct from its siblings.
+- `lavish`: Apply the same amount of vertical space as padded [containers](/?path=/docs/objects-container--basic) have for consistent spacing across separate sections.
 
 <Canvas>
-  <Story
-    name="Compact"
-    parameters={{ docs: { source: { code: listDemoSource } } }}
-  >
-    {listDemo}
+  <Story name="Amount" args={{ amount: 'condensed' }}>
+    {(args) => demo(args)}
   </Story>
 </Canvas>
 
-While `o-rhythm--generous` is intended for separating distinct content chunks with more whitespace:
+## Heading Amount
 
-<Canvas>
-  <Story
-    name="Generous"
-    parameters={{ docs: { source: { code: paragraphsDemoSource } } }}
-  >
-    {paragraphsDemo}
-  </Story>
-</Canvas>
+It can be helpful to differentiate headings with more preceding space than normal flow content. In those cases, you can specify a `heading_amount` property or a class name of `o-rhythm--{heading_amount}-headings` with one of the following keywords:
 
-And finally, `o-rhythm--lavish` can separate entire sections from each other. It uses the same fluid padding values as [the container object](/?path=/docs/objects-container--basic):
+- `generous`
+- `lavish`
 
 <Canvas>
   <Story
-    name="Lavish"
-    parameters={{
-      layout: 'fullscreen',
-      docs: { source: { code: sectionsDemoSource } },
-    }}
+    name="Heading Amount"
+    args={{ amount: 'condensed', heading_amount: 'generous' }}
   >
-    {sectionsDemo}
+    {(args) => demo(args)}
   </Story>
 </Canvas>
 
-To revert to the default spacing in nested rhythm objects, use the `o-rhythm--default` modifier.
+## Inheriting Rhythm
 
-## Heading Modifiers
+The vertical spacing applied by the Rhythm object only applies to direct children. In some cases, you may want elements below to inherit rhythm. This can be accomplished in two main ways.
 
-The `o-rhythm--generous-headings` modifier helps differentiate sections of articles (or similarly long prose content) by adding more whitespace before headings (or elements with a class name containing `heading`, such as [our heading component](/docs/components-heading--levels#heading)).
+### Nesting Rhythm Objects
 
-<!--
-This example isn't inlined so any media queries affecting the container will
-behave as expected.
--->
+Nested rhythm objects will inherit any spacing set via modifiers. In this example, the first list inherits the parent element's rhythm while the second resets it using the `default` amount:
```

Comment:
> In this example, the first list inherits the parent element's rhythm while the second resets it using the default amount:

I only see one list. Am I misunderstanding? 🤔

Reply:
Ah, good catch. At the last minute I realized that the second list in the example made it seem more complicated than it was, so I removed it, but forgot to update the preceding copy.

```suggestion
Nested rhythm objects will inherit any spacing set via modifiers. In this example, the unordered list is also a Rhythm object. It inherits the parent object's spacing even though no modifier is set:
```